modules/node-certificates.adoc

```diff
@@ -0,0 +1,29 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="node-certificates_{context}"]
+= Node certificates
+
+== Purpose
+
+The Kubelet, in 4.x, uses the bootstrap certificate located in
+`/etc/kubernetes/kubeconfig` to initially bootstrap. This is followed by the
+link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#bootstrap-initialization[bootstrap
+initialization process] and
+link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#authorize-kubelet-to-create-csr[authoization
+of the kubelet to create a CSR].
+
+In that process, The kubelet generates a CSR while communicating over the
+bootstrap channel. The controller manager signs the CSR, resulting in a
+certificate  that the kubelet manages.
+
+== Expiration
+This bootstrap certificate is valid for 10 years.
```

Comment:
The "certificate" here should be called a CA (this certificate is a Certificate Authority) used to sign/approve node certificates.